modules/integration/tests-common/integration-test-utils/src/main/java/org/wso2/am/integration/test/utils/bean/APIManageBean.java

```diff
@@ -0,0 +1,284 @@
+/*
+ *Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *WSO2 Inc. licenses this file to you under the Apache License,
+ *Version 2.0 (the "License"); you may not use this file except
+ *in compliance with the License.
+ *You may obtain a copy of the License at
+ *
+ *http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *Unless required by applicable law or agreed to in writing,
+ *software distributed under the License is distributed on an
+ *"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *KIND, either express or implied.  See the License for the
+ *specific language governing permissions and limitations
+ *under the License.
+ */
+
+package org.wso2.am.integration.test.utils.bean;
+
+public class APIManageBean extends AbstractRequest {
+    private String name;
+    private String type = "";
+    private String version = "";
+    private String provider = "";
+    private String default_version_checked = "";
+    private String transport_https = "";
+    private String transport_http = "";
```

Comment:
shall we use camel case for all variables in here

Reply:
Form params are sent exactly as above (with underscores), from UI. We can still change these as they carry only the value, but keeping consistent everywhere might make more clear. WDYT ?